velox/functions/lib/string/StringImpl.h

```diff
@@ -197,38 +197,110 @@ std::vector<int32_t> stringToCodePoints(const T& inputString) {
   return codePoints;
 }
 
-/// Returns the starting position in characters of the Nth instance(counting
-/// from the left if lpos==true and from the end otherwise) of the substring in
-/// string. Positions start with 1. If not found, 0 is returned. If subString is
-/// empty result is 1.
+/// Returns the starting position in characters of the Nth instance of the
+/// substring in string after 'startPosition'. Positions start with 1. If not
+/// found, 0 is returned. If subString is empty result is 1.
+/// @tparam lpos If true, counting from the left and from the end otherwise.
```

Comment:
> from the left and from the end

'left' doesn't match 'end'; perhaps, "If true, counting from the start of the string. Counting from the end of the string otherwise."

Reply:
Updated. Thanks.